srcpkgs/mir/template

```diff
@@ -0,0 +1,232 @@
+# Template file for 'mir'
+pkgname=mir
+version=0.31.2
+revision=1
+build_style=cmake
+configure_args="-DMIR_ENABLE_TESTS=OFF"
+hostmakedepends="pkg-config"
+makedepends="MesaLib-devel boost-devel capnproto-devel freetype-devel
+ gflags-devel glm glog-devel libatomic-devel libepoxy-devel libinput-devel
+ libxkbcommon-devel libxml++-devel lttng-ust-devel nettle-devel protobuf-devel
+ python3-Pillow wayland-devel"
+depends="dbus qt5-wayland"
+short_desc="Cross-platform display manager"
+maintainer="maxice8 <thinkabit.ukim@gmail.com>"
+license="GPL-3.0-only"
+homepage="https://mir-server.io/"
+distfiles="https://github.com/MirServer/mir/releases/download/v${version}/mir-${version}.tar.xz"
+checksum=a4be0c36c9cd0c91f966c4d70ac21e916758adcbf2523ded71a672938a65bad3
+nocross="src/protobuf/CMakeFiles/mirprotobuf.dir/build.make:61: *** target pattern contains no '%'. "
+
+case "$XBPS_TARGET_MACHINE" in
+	*-musl) broken="dlvsym usage" ;;
```

Comment:
`dlvsym` is mostly easy to patch by replacing it with `dlsym` which is mostly sufficient.
But they are also using a few GLIBC only pthread stuff, which I don't know if it iwll be easy to patch. Will check it tomorrow.